Dockerfile

```diff
@@ -19,20 +21,18 @@ COPY version/ version/
 ARG TARGETARCH
 ARG VERSION
 ARG GITVERSION
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} GO111MODULE=on \
-    go build -a -ldflags "-X github.com/oam-dev/kubevela/version.VelaVersion=${VERSION:-undefined} -X github.com/oam-dev/kubevela/version.GitRevision=${GITVERSION:-undefined}" \
+RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} \
+    go build -a -ldflags "-s -w -X github.com/oam-dev/kubevela/version.VelaVersion=${VERSION:-undefined} -X github.com/oam-dev/kubevela/version.GitRevision=${GITVERSION:-undefined}" \
     -o manager-${TARGETARCH} main.go
 
-# Use ubuntu as base image for convenience.
+# Use alpine as base image to reduce image size
 # You can replace distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 # Could use `--build-arg=BASE_IMAGE=gcr.io/distroless/static:nonroot` to overwrite
 ARG BASE_IMAGE
-FROM ${BASE_IMAGE:-ubuntu:latest}
+FROM ${BASE_IMAGE:-alpine:latest}
```

Comment:
Here is the reason why we change the base image, https://github.com/oam-dev/kubevela/issues/1177

Does this change fit for this issue?

Reply:
If we need to solve #1177 better, support entrypoint and debugging like `docker run -it --rm vela-core:latest sh` at the same time, we need to add an `entrypoint.sh` file. So both `docker run -it --rm vela-core:latest -h` and `docker run -it --rm vela-core:latest sh` can run well

Reply:
I will make this change

Reply:
Done